### PR TITLE
lint: duplicate import

### PR DESCRIPTION
**Pull request by nektro**
found with https://github.com/nektro/ziglint and manually updated

### CODE REVIEW

File: lib/std/special/compiler_rt/int.zig [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // Builtin functions that operate on integer types
 const builtin = @import("builtin");
-const testing = @import("std").testing;
-const maxInt = @import("std").math.maxInt;
-const minInt = @import("std").math.minInt;
+const std = @import("std");
+const testing = std.testing;
+const maxInt = std.math.maxInt;
+const minInt = std.math.minInt;
```
Comment (marler8997) [inline lib/std/special/compiler_rt/int.zig:6]: So this is considered bad?

Comment (nektro) [reply lib/std/special/compiler_rt/int.zig:6]: not inherently, but i think it helps readability

File: src/main.zig [hunk 2]
```diff
@@ -23,6 +23,10 @@ const Cache = @import("Cache.zig");
 const target_util = @import("target.zig");
 const ThreadPool = @import("ThreadPool.zig");
 const crash_report = @import("crash_report.zig");
+const Module = @import("Module.zig");
+const AstGen = @import("AstGen.zig");
+const Zir = @import("Zir.zig");
+const llvm = @import("codegen/llvm/bindings.zig");
```
Comment (andrewrk) [inline src/main.zig:29]: I don't want these in scope for the rest of the file. Don't want any accidental dependencies on these namespaces.

File: test/behavior/import.zig [hunk 3]
```diff
@@ -12,7 +13,7 @@ test "importing the same thing gives the same import" {
 
 test "import in non-toplevel scope" {
     const S = struct {
-        usingnamespace @import("import/a_namespace.zig");
+        usingnamespace a_namespace;
```
Comment (andrewrk) [inline test/behavior/import.zig:16]: notice the name of the test

**Review by andrewrk — CHANGES_REQUESTED** [review]
Overall looks good- thank you for doing the lint. Just a couple fixes and then this is good to go.

**Pull request merged**